index.d.ts

```diff
@@ -440,7 +441,7 @@ declare namespace Moleculer {
 
 	type ActionVisibility = "published" | "public" | "protected" | "private";
 
-	type ActionHookBefore = (ctx: Context<any, any>) => Promise<void> | void;
+	type ActionHookBefore = (ctx: Context<any, any>) => Promise<any> | any;
```

Comment:
@icebob Can before hooks return anything?  Its not documented that they can (after hooks explicitly say they can return something).  If I'm understanding `action-hooks.js` properly, it doesn't seem like the value resolved from a before hook would do anything at all.  I'd hesitate to make this change unless there is a specific reason for returning a value from a before hook.

Reply:
Yeah, it has no effect, return value is not processed.